<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>etedit</class>
 <widget class="QDialog" name="etedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#etedit {
background-color: qlineargradient(spread:reflect, x1:0.0452273, y1:0.704, x2:1, y2:0, stop:0 rgba(30, 137, 174, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qlineargradient(spread:reflect, x1:0.0452273, y1:0.704, x2:0.972, y2:0.0568182, stop:0.142045 rgba(30, 137, 174, 255), stop:0.9375 rgba(255, 255, 255, 255));
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>122</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
font-weight:bold;
padding-left: 5px;
padding-right: 5px;
min-width: 110px;
max-width: 110px;
min-height: 20px;
max-height: 20px;</string>
   </property>
   <property name="text">
    <string>Choose Employee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>112</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
font-weight:bold;
padding-left: 5px;
padding-right: 5px;
min-width: 100px;
max-width: 100px;
min-height: 20px;
max-height: 20px;</string>
   </property>
   <property name="text">
    <string>Choose Training</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
font-weight:bold;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width: 70px;
min-height: 20px;
max-height: 20px;</string>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
font-weight:bold;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width: 70px;
min-height: 20px;
max-height: 20px;</string>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
font-weight:bold;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width: 70px;
min-height: 20px;
max-height: 20px;</string>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ec">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_tc">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_sd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_ed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_status">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_et_save">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
font-weight:bold;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width: 70px;
min-height: 20px;
max-height: 20px;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
